_includes/layout/base/pagetop-donation.html

```diff
@@ -0,0 +1,71 @@
+{% comment %}
+This file is licensed under the MIT License (MIT) available on
+http://opensource.org/licenses/MIT.
+{% endcomment %}
+
+<div class="donation-container">
+  <div class="donation-text">
+    <p>
+      {% translate donation-banner-text layout %}
+    </p>
+    <button class="donation-btn" onclick="openDonationModal()">
```

Comment:
Actually, type submit is required in forms since they are the only element for which a submit action makes sense.
Outside of forms, buttons should be just buttons.
Internet Explorer (10- or 9-) will trigger the first submit button found in DOM when pressing enter in an input that is outside of a form.